test/test_torch.py

```diff
@@ -4298,8 +4298,9 @@ def test_symeig(self):
         self.assertEqual(cov, ahat, 1e-8, 'VeV\' wrong')
 
         # test eigenvectors=False
-        rese2 = torch.zeros(3)
-        resv2 = torch.randn(3, 3)
+        cov = cov.cuda()
+        rese2 = torch.zeros(3).cuda()
+        resv2 = torch.randn(3, 3).cuda()
```

Comment:
You could probably make this test function a static function, and then call this separately in `test_torch` and `test_cuda`. Have a look at `test_det_logdet_slogdet` / `test_pinverse` for reference.